packages/react-pose/src/components/Transition/children.ts

```diff
@@ -100,7 +100,7 @@ const handleTransition = (
       }
 
       const nextIndex = nextKeys.indexOf(key);
-      return nextIndex !== -1 && i !== nextIndex;
+      return nextIndex !== -1;
```

Comment:
Is the resulting code any different to `leaving.indexOf(key) !== -1` basically saying, if it isnt entering or leaving?